tests/test_get_next_results.py

```diff
@@ -13,12 +13,15 @@ def test_get_next_results(base_url, selenium):
     assert len(page.result_sets) == 10
 
     page.get_next_ten_results()
+    assert page.results_visible
```

Comment:
I don't remember why I added this in the first place as there are waits already embedded in the get_next methods- updated and the test passes repeatedly without results_visible.